src/prefect/collection_blocks_data.json

```diff
@@ -1,5 +1,977 @@
 {
```

Comment:
Can we move this file into the `orion` module?

I also noticed we're using `aiofiles` to load this? Not sure when/why that became a dependency, let's just use `anyio.open` and remove that dependency.